examples/test/test_i_one_kms_cmk.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This should be "Copyright 2019", not "2017-2019".

The year/range defines two things: when the file was created and when it was most recently edited.
If those years are the same, we do not need a range.